data/VaradBelwalkar.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "Varad Belwalkar",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "DevOps and Machine Learning Enthusiast",
+	"avatar": "https://github.com/VaradBelwalkar.png",
```

Comment:
`avatar` is not a required field anymore

Reply:
https://linkfree.io/docs/full-profile-example. Check the example json file here and make changes. Also can go on the playground https://linkfree.io/playground to verify if you have a valid json after making changes

Reply:
Great! will make those changes!